### PR TITLE
refactor(itemCalculator): split calculators into their own files

### PR DESCRIPTION
**Pull request by DuckySoLucky**

### CODE REVIEW

File: calculators/items/HotPotatoBook.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const { APPLICATION_WORTH } = require('../../constants/applicationWorth');
+
+class HotPotatoBookCalculation {
+    constructor({ calculation, itemData, prices, price }) {
+        this.calculation = calculation;
+        this.itemData = itemData;
+        this.prices = prices;
+        this.price = price;
+        // this.calculate();
```
Comment (DuckySoLucky) [inline calculators/items/HotPotatoBook.js:9]: Tehnically we could call this, so there would be no need for [this](https://github.com/Altpapier/SkyHelper-Networth/blob/revamp-duckysolucky/calculators/ItemNetworthCalculator.js#L113-L116).

File: calculators/ItemNetworthCalculator.js [hunk 2]
```diff
@@ -23,29 +23,34 @@ const { reforges } = require('../constants/reforges');
 const { getHypixelItemInformationFromId } = require('../constants/itemsMap');
 
 const networthManager = require('./NetworthManager');
+const { Helper } = require('./helper/functions');
+const { HotPotatoBookCalculation } = require('./items/HotPotatoBook');
+const { Recombobulator3000Calculation } = require('./items/Recombobulator_3000');
+const { PickonimbusCalculation } = require('./items/Pickonimbus');
 
-class ItemNetworthCalculator {
+class ItemNetworthCalculator extends Helper {
```
Comment (MattTheCuber) [inline calculators/ItemNetworthCalculator.js:31]: Why extend `Helper`? What else is `Helper` used for?

File: calculators/ItemNetworthCalculator.js [hunk 3]
```diff
@@ -77,168 +82,51 @@ class ItemNetworthCalculator {
...
-                id: 'RECOMBOBULATOR_3000',
-                type: 'recombobulator_3000',
-                price: (prices['recombobulator_3000'] || 0) * recombApplicationWorth,
-                count: 1,
-            };
-            this.price += calculationData.price;
-            this.calculation.push(calculationData);
+        const calculations = [HotPotatoBookCalculation, PickonimbusCalculation, Recombobulator3000Calculation];
+        for (const Calculation of calculations) {
+            Calculation.calculate(this);
```
Comment (MattTheCuber) [inline calculators/ItemNetworthCalculator.js:115]: I like this as a static method call like is here 👍 

Would also need to pass prices ofc.

File: calculators/items/HotPotatoBook.js (hunk 1 above)
Comment (MattTheCuber) [reply calculators/items/HotPotatoBook.js:9]: Yeah, no need for a constructor. Harder to maintain and not need if we use static methods.

**Review by MattTheCuber — COMMENTED** [review]
looks great. now we need to decide! XD

File: calculators/ItemNetworthCalculator.js (hunk 2 above)
Comment (DuckySoLucky) [reply calculators/ItemNetworthCalculator.js:31]: Mostly to clean out the main fie, it has getItemid(), getItemName() and basic checks

https://github.com/Altpapier/SkyHelper-Networth/blob/revamp-duckysolucky/calculators/helper/functions.js

**Pull request closed without merging**